release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -469,6 +469,13 @@ The following metric has been removed:
 
 * `ovnkube_master_skipped_nbctl_daemon_total`
 
+[id="ocp-4-12-multi-zone-ipi-vsphere-installation"]
+==== Multi-zone Installer Provisioned Infrastructure VMware vSphere installation (Technology Preview)
+Beginning with {product-title} {product-version}, 
+the ability to configure multiple vCenter datacenters and multiple vCenter clusters in a single vCenter installation using installer-provisioned infrastructure is now available as a Technology Preview feature. Using vCenter tags, you can use this feature to associate vCenter datacenters and compute clusters with openshift-regions and openshift-zones. These associations can help separate failure domains.
```

Comment:
The last line still reads funny:
These associations can help separate failure domains.

How about:
These associations defines failure domains to enable application workloads to be associated with specific locations and/or failure domains.